src/Features/CSharp/Portable/UsePatternMatching/CSharpAsAndNullCheckDiagnosticAnalyzer.cs

```diff
@@ -373,33 +408,36 @@ private static SyntaxNode GetLeftmostCondition(SyntaxNode node)
         {
             while (true)
             {
-                switch (node?.Kind())
```

Comment:
💭 This seems like an unnecessary change

Reply:
Relates to https://github.com/dotnet/roslyn/issues/21962

I can revert it if you want.